### ISSUE TITLE
Drag and drop support

### ISSUE THREAD

**Issue by boydgreenfield** [onecodex/onecodex-uploader#6]
- [ ] Mac OS X
- [x] Windows

**Comment by bovee** [comment]
Drag and drop does work on Windows now, but only takes one of the files (and may display an error for every file that's not a proper sequencing file). Think we should still hold off on publicizing the availability of drag and drop until we have:
 - [ ] a functional drag and drop solution for Mac OS X (converting the urls into valid file paths)
 - [ ] multiple file upload (issue #6 )
It may also be nice to have drag and drop onto the icon on the dock, but that might have to wait on https://github.com/pyinstaller/pyinstaller/issues/1309
